static/app/views/issueList/overview.tsx

```diff
@@ -208,6 +208,10 @@ class IssueListOverview extends React.Component<Props, State> {
       }
     }
 
+    if (prevState.itemsRemoved !== this.state.itemsRemoved) {
+      this.fetchData();
```

Comment:
i'd rather call this directly in onMarkReviewed if we can replace all state updates by calling it directly

Reply:
changed the way this works (see other comment)